knowledge-base/autero1/action-gotestsum/action-security.yml

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
@arjundashrath please remove the extra space at the top

Reply:
Done